ui/partials/footer-content.hbs

```diff
@@ -1,6 +1,14 @@
 <footer class="footer">
   <p>Copyright (C) 2019 Hercules Labs</p>
 </footer>
+<script src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
```

Comment:
I don't think we should depend on too many CDNs. Not a problem now, but slippery slope.

Reply:
Not much of a choice here, I'm 100% sure their api and js/css need to be in sync.